cva6/tests/custom/isacov/branch_to_zero.S

```diff
@@ -34,12 +34,21 @@ branch:
   c.bnez     s1, branch
 branch1:
   c.beqz     s0, branch1
+branch2:
```

Comment:
How does the code break out of this loop?  Is there a timer-based trap in the BSP that changes `t5`?

Reply:
To cover immb = 0x0 I give a false condition (isn't like jump); so if we give a s0 != zero we gonna cover immb = 0x0 because it's been decoded also we don't get a infinite loop.

Reply:
So where does `s0` get set?

Reply:
The code makes sense to me, as long as you initialize all the registers that are used for comparison (so I am also missing an `s0` initialization here)